.circleci/config.yml

```diff
@@ -0,0 +1,27 @@
+version: 2.1
+jobs:
+    build:
+        docker:
+            - image: circleci/node:14-browsers
+        steps:
+            - checkout
+            - run:
+                  name: Update npm
```

Comment:
Should this also be cached?

Reply:
Not going to address right now since the CI is not being slowed significantly by this. Split into new issue: #49